rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverGlobalLayoutEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.jakewharton.rxbinding.view;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.View;
+
+public final class ViewTreeObserverGlobalLayoutEvent extends ViewEvent<View> {
```

Comment:
I'm wondering whether it's useful to actually have an event object for this? Or should we just emit an `Object` instance as a signal (like with clicks)?